src/BuildEnv/Build.hs

```diff
@@ -389,21 +395,30 @@ buildPlan verbosity comp fetchDir0 destDir0
         unitSetupScript pu@(ConfiguredUnit { puSetupDepends }) = do
           let pkgDir = getPkgDir fetchDir pu
           setupPackage verbosity comp pkgDbDirs pkgDir puSetupDepends
-        unitBuildScript :: ConfiguredUnit -> BuildScript
-        unitBuildScript pu =
+        unitBuildScript :: Args -> ConfiguredUnit -> BuildScript
+        unitBuildScript extraConfigureArgs pu@(ConfiguredUnit { puId }) =
           let pkgDir = getPkgDir fetchDir pu
+              mbEventLogArg = mbEventLogDir <&> \ logDir ->
+                let
+                  logPath = logDir </> "ghc" <.> Text.unpack (unUnitId puId) <.> "eventlog"
+                in "--ghc-option=\"-with-rtsopts= -l -ol" <> logPath <> "\""
```

Comment:
This doesn't seem to work... I'm not sure what I'm doing wrong, but I'm not getting any eventlogs in the output directory when doing this?